lib/dialects/sqlite3/schema/ddl.js

```diff
@@ -308,10 +308,17 @@ class SQLite3_DDL {
               if (defLine.includes(indexName)) return false;
               return true;
             } else {
-              const matched = defLine.match(/\(`(\S+)`\)/);
-              const columnName = matched[1];
-
-              return columns.includes(columnName) === false;
+              const matched = defLine.match(/\(`([^)]+)`\)/);
+              const columnNames = matched[1].split(', ');
+
+              let unknownColumnIncluded = false;
+              for (let i = 0; i < columnNames.length; i++) {
```

Comment:
Wouldn't implementation based on `some` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) be simpler?

Reply:
You are right that would be simpler. But when I look at the code snippet I think the check should be performed  with Array.every, because all columns of the FK are relevant, so my fault.